examples/webgl_multiple_elements_text.html

```diff
@@ -275,7 +277,7 @@
 
 				} );
 
-				t ++;
+				t += clock.getDelta() * 30;
```

Comment:
Really small nit but if we want the animation in the examples to run at the same rate as they would have previously on a common 60hz display this should `60` not `30`. The animations now run slower than they did, though it's inconsequential:

https://raw.githack.com/CalebBabin/three.js/dev/examples/webgl_multiple_elements_text.html

Reply:
Updated!